packages/hint-create-element-svg/src/svgElements.ts

```diff
@@ -0,0 +1,73 @@
+const svgElements: string[] = [
```

Comment:
Question: What is faster, ask use 'includes' in an array, or `has` in a Set?, if set is faster, I think we should change this to a Set.

Reply:
Is it Friday yet? Duh! Yes, set is for sure the way to go here for the lookups.